CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Table of Contents
+
+* [Welcome!](#welcome)
+* [Contributing to Scrapers](#contributing-to-scrapers)
+	* [General Guidelines](#general-guidelines)
+	* [Minimum Requirements for Submission](#minimum-requirements-for-submission)
+    * [Testing and Validation](#testing-and-validation)
+    * [Getting Help](#getting-help)
+
+# Welcome!
+
+Welcome, and thank you for your interest in contributing to the Police Data Accessibility Project!
+
+There are a number of ways you can help contribute to the project. We need assistance with areas such as Scraper Development, Legal, DevOps, FOIA Requests, and Data Architecture. If you are interested in getting involved with the project beyond Scraper Development (which is the focus of this contribution guideline), please join our Slack channel - {include link here}
+
+NOTE: We are on a free Slack instance, and the link expires every three weeks. If you attempt to join and the link is invalid, please open an issue and someone will have a new link up shortly. 
+
+# Contributing to Scrapers
+
+## General Guidelines
+
+The majority of scraper code for this project is currently being written in Python. While we are open to conversations about other languages, there is a desire to standardize as much as possible for simplicity of administration and consistency. 
+
+Given the nature of this project, there are some extremely important guidelines below that go along with contributing to web scraping. 
+
+1. Your scraper must be anonymizing data that it gathers, removing identifiable information to prevent disclosure of names of arresting officers or cited citizens.
```

Comment:
This is very important. It might be worthwhile figuring out a way to get the anonymizing logic out of the scrapers. 

I'm thinking kind of like how log aggregation works. Scraped data get's JSONified and pushed into a common staging area. Some other process comes in to sanitize the data and push it into whatever the final database is.

Reply:
We still do want arresting officer badge number though. Some discussions on slack proposed hashing the badge numbers before making public. We do want to be able to have an individual cop level data point that we scrape for all counties though.